### PR TITLE
Add license metadata to pg19

### PR DESCRIPTION
**Pull request by julien-c**
As reported over email by Roy Rijkers

### CODE REVIEW

**Review by albertvillanova — APPROVED** [review]
Thanks Roy and Julien for the addendum.

File: datasets/pg19/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 language:
 - en
+license: apache-2.0
```
Comment (albertvillanova) [inline datasets/pg19/README.md:4]: The license tag should be an array.

**Review by albertvillanova — CHANGES_REQUESTED** [review]
Fix needed to pass the validation.

File: datasets/pg19/README.md (hunk 1 above)
Comment (julien-c) [reply datasets/pg19/README.md:4]: hmm that's not what's documented on https://huggingface.co/docs/hub/datasets-cards =) 

Screenshot 2022-08-11 at 16 12 20

Comment (julien-c) [reply datasets/pg19/README.md:4]: but ok i'll push this for consistency

**Review by albertvillanova — APPROVED** [review]
Then non-passing tests have nothing to do with this PR, but with other missing information in the dataset card.

File: datasets/pg19/README.md (hunk 1 above)
Comment (albertvillanova) [reply datasets/pg19/README.md]: Good catch: we should align docs and validation implementation:

https://github.com/huggingface/datasets/blob/cfae8ac7c896772af537441508aaae2391aea53a/src/datasets/utils/metadata.py#L232

**Pull request merged**